src/shared-components/container/__snapshots__/test.tsx.snap

```diff
@@ -99,7 +99,7 @@ exports[`Container UI snapshots renders message type container 1`] = `
   background-color: #FFFFFF;
   border: 1px solid #EDEDF0;
   border-radius: 16px;
-  box-shadow: 0 12px 20px 0 rgba(51,46,84,0.05);
+  box-shadow: 0px 8px 24px rgba(52,51,82,0.05);
```

Comment:
message to clickable refactor. Not much change visually

Reply:
This seems fine, especially in light of the visual change not being very significant. 

However, applying a `clickable` box-shadow to things that are not clickable seems like a smell to me. Re-investing in accessibility after we launch Agency will be a major focus, and I'm concerned about inconsistency in code usage and our styleguide if we are providing engineers with designs that use `clickable` styling on non-clickable things. A major focus will be using the correct HTML, full keyboard navigation, etc. I don't want to obfuscate what is clickable (and the controls we're required to provide for things that are) and the appropriate associated styling. 

I don't want this to be a blocker for this PR, but would like this clarified, so I'm cc'ing @benkolde. As an aside, it also looks like we've got a copy-paste error for `clickable`, since I don't think the `modal` usage description duplication is intended. 

<img width="329" alt="Screen Shot 2020-12-07 at 3 03 13 PM" src="https://user-images.githubusercontent.com/13544620/101405188-5a3e8600-389d-11eb-96e8-2e9e17f07c2b.png">

I could see usage of this being annoying, e.g. individual engineers asking why we're applying `clickable` to non-clickable elements.  and this  be applying clickable box-shadow styling to non-clickable elements, but if it's fine with Design (especially in light of the change being not too noticeable)

Reply:
Ben said we'll deal with this later
https://www.notion.so/curology/BOX_SHADOWS-Analysis-c4bc7757eeab472ca13e4752428957c4